StyleCop.Analyzers/StyleCop.Analyzers.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -83,15 +80,15 @@ protected Task VerifyCSharpDiagnosticAsync(string[] sources, DiagnosticResult[]
         /// </summary>
         /// <param name="sources">An array of strings to create source documents from to run the analyzers on.</param>
         /// <param name="language">The language of the classes represented by the source strings.</param>
-        /// <param name="analyzer">The analyzer to be run on the source code.</param>
+        /// <param name="analyzers">The analyzers to be run on the source code.</param>
         /// <param name="expected">A collection of <see cref="DiagnosticResult"/>s that should appear after the analyzer
         /// is run on the sources.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> that the task will observe.</param>
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
-        private async Task VerifyDiagnosticsAsync(string[] sources, string language, DiagnosticAnalyzer analyzer, DiagnosticResult[] expected, CancellationToken cancellationToken)
+        private async Task VerifyDiagnosticsAsync(string[] sources, string language, ImmutableArray<DiagnosticAnalyzer> analyzers, DiagnosticResult[] expected, CancellationToken cancellationToken)
         {
-            var diagnostics = await GetSortedDiagnosticsAsync(sources, language, analyzer, cancellationToken).ConfigureAwait(false);
-            VerifyDiagnosticResults(diagnostics, analyzer, expected);
+            var diagnostics = await GetSortedDiagnosticsAsync(sources, language, analyzers.ToImmutableArray(), cancellationToken).ConfigureAwait(false);
```

Comment:
:bulb: The call to `ToImmutableArray()` is unnecessary here (I removed it when I merged this pull request).